WordPress/Classes/System/WordPressAppDelegate+Swift.swift

```diff
@@ -93,6 +93,24 @@ extension WordPressAppDelegate {
 
         return WPTabBarController.sharedInstance().currentlySelectedScreen()
     }
+
+    @objc func isWelcomeScreenVisible() -> Bool {
```

Comment:
Now it's in Swift, I'd probably change this to a computed `var` instead of a `func`.

Reply:
I actually didn't think a computed var would work with `@objc`. Nice suggestion :D